### PR TITLE
[hi] Add content/hi/includes/task-tutorial-prereqs.md

### PR DESCRIPTION
**Pull request by Nawaz027**
Added task-tutorial-prereqs on the path website/content/hi/includes.
Please have a look @anubha-v-ardhan

### CODE REVIEW

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/includes/task-tutorial-prereqs.md

**Comment by anubha-v-ardhan** [comment]
fixes #30590

File: content/hi/includes/task-tutorial-prereqs.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। ये अनुशंसित किया गया है की इस अनुशीक्षण को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो के कंट्रोल प्लेन होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप मिनिक्यूब की मदद से बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का इस्तेमाल कर सकते है :
```
Comment (anubha-v-ardhan) [inline content/hi/includes/task-tutorial-prereqs.md:1]:
```suggestion
आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सकें। हमारी सलाह है की इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो कि कंट्रोल प्लेन 
होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप 
[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
की मदद से वह बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का इस्तेमाल कर सकते हैं:

```

File: content/hi/includes/task-tutorial-prereqs.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। ये अनुशंसित किया गया है की इस अनुशीक्षण को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो के कंट्रोल प्लेन होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप मिनिक्यूब की मदद से बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का इस्तेमाल कर सकते है :
+•	काटाकोडआ
+•	प्ले विद कुबरनेट्स
```
Comment (anubha-v-ardhan) [inline content/hi/includes/task-tutorial-prereqs.md:3]:
```suggestion
* [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
* [प्ले विथ कुबेरनेट्स](http://labs.play-with-k8s.com/)
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Links should not be removed

File: content/hi/includes/task-tutorial-prereqs.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/includes/task-tutorial-prereqs.md:1]: Feel free to use new line when needed, markdown doesn't mind and it makes it more easily readable

**Comment by Nawaz027** [comment]
please have a look @anubha-v-ardhan

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @Nawaz027

**Pull request merged**